sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -125,8 +125,13 @@ object JdbcUtils extends Logging {
       dialect: JdbcDialect): Iterator[Byte] = {
     val conn = getConnection()
     var committed = false
+    var supportsTransactions = true
```

Comment:
Why a var, the initialization, or following try block? seems like it's one line to init a val.